app/assets/javascripts/modules/media_viewer.js

```diff
@@ -98,42 +157,23 @@
           var checkWindow = setInterval(function() {
             if (!_timedOut(start, 30000) &&
               (!windowReference || !windowReference.closed)) return;
-            clearInterval(checkWindow);
-            mediaObject
-              .parents(sliderObjectSelector)
-              .find('[data-auth-link], ' + restrictedOverlaySelector + ', ' + restrictedMessageSelector)
-              .hide();
-            // As best I can tell, .touch() does nothing (and does not exist in jQuery),
-            // but without calling some function after .load() the media fails to load.
-            mediaObject[0].load().touch();
+            authCheckForMediaObject(mediaObject, function(_, data) {
+              if(data.status === 'success') {
+                clearInterval(checkWindow);
+                mediaObject
+                  .parents(sliderObjectSelector)
+                  .find('[data-auth-link], ' + restrictedOverlaySelector + ', ' + restrictedMessageSelector)
+                  .hide();
```

Comment:
The things being hidden in 128-134 are overlays and divs with message text.  This particular code is hiding the Login link itself.

Now that I look at this though, we should have a direct reference to the link (since this is in an on-click event) and shouldn't have to re-find it.  I'll see how easy that is to change.